.travis.yml

```diff
@@ -1,9 +1,13 @@
+dist: bionic
+addons:
+  apt:
+    packages:
+      - openjdk-8-jre
+
 language: scala
 scala:
   - 2.11.12
   - 2.12.6
-jdk: oraclejdk8
```

Comment:
Just replacing this with `openjdk8` should work?
https://github.com/nevillelyh/shapeless-datatype/blob/master/.travis.yml#L7

Reply:
thank you!! really appreciate the example @nevillelyh ! https://github.com/spotify/ratatool/pull/164 is looking like it's passing so I will close this in favor of that